src/components/Navbar/NavbarLink.tsx

```diff
@@ -28,14 +28,16 @@ export const NavbarLink: FC<NavbarLinkProps> = ({
   children,
   className,
   theme: customTheme = {},
+  onClick: userOnClick,
```

Comment:
No need to rename. Use just `onClick`

Reply:
I've just renamed on differ the actual DOM's `onClick` event, @nigellima , but let me rename it to original

Reply:
Excellent. Thanks @dhavalveera